test/distributed/test_c10d_common.py

```diff
@@ -596,6 +596,76 @@ def forward(self, x):
         return F.softmax(self.embedding(x), dim=1)
 
 
+class AbstractProcessGroupWrapperTest(MultiProcessTestCase):
```

Comment:
nit: I'm wondering whether we can put all the tests into a separate file 'test_process_group_wrapper.py'?